### PR TITLE
add default SignTransactionModal for extra provider

### PR DESCRIPTION
**Pull request by StanislavSava**

### CODE REVIEW

File: src/UI/SignTransactionsModals/SignWithExtraModal/index.tsx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+import { useSignTransactions } from 'hooks/transactions/useSignTransactions';
+
+export function SignWithExtraModal() {
+  useSignTransactions();
+  return <div />;
```
Comment (arhtudormorar) [inline src/UI/SignTransactionsModals/SignWithExtraModal/index.tsx:6]: return null?

**Review by arhtudormorar — APPROVED** [review]

File: src/UI/SignTransactionsModals/SignWithExtraModal/index.tsx (hunk 1 above)
Comment (StanislavSava) [reply src/UI/SignTransactionsModals/SignWithExtraModal/index.tsx:6]: ts cries, are nevoie de Element

Comment (CiprianDraghici) [reply src/UI/SignTransactionsModals/SignWithExtraModal/index.tsx:6]: Maybe </>

**Review by CiprianDraghici — APPROVED** [review]

**Comment by CiprianDraghici** [comment]
Update changelog "Unreleased" section and it's ready to merge

**Pull request merged**